Eevee.Sleep.Bot/Controllers/Mongo/InGameAnnouncement/OfficialSite/OfficialsiteAnnoucementHistoryControllers.cs

```diff
@@ -0,0 +1,41 @@
+using Eevee.Sleep.Bot.Models.InGameAnnouncement.OfficialSite;
+using MongoDB.Driver;
+
+namespace Eevee.Sleep.Bot.Controllers.Mongo.InGameAnnouncement.OfficialSite;
+
+public static class OfficialSiteAnnouncememntHistoryController {
```

Comment:
We were dealing with a different model, so I didn't dare generalize.
I figured it was better for me not to generalize because I think there could be other logic mixed in.

Reply:
But if it's different model with the same interface, it should be fine generalizing them (at least `Upsert` and `BulkUpsert` should have the same behavior)